src/site/xdoc/server/config.xml

```diff
@@ -128,6 +128,11 @@
         <td><a href="config-jmap.html">JMAP Configuration (Guice only)</a></td>
         <td></td>
       </tr>
+      <tr>
+        <td><a href="https://github.com/apache/james-project/server/container/guice/memory-guice/sample-configuration/webadmin.properties">webadmin.properties</a></td>
```

Comment:
Not found :trollface: 

`https://github.com/apache/james-project/blob/master/server/container/guice/memory-guice/sample-configuration/webadmin.properties`

maybe better to use this, it contains many options :)

`https://github.com/apache/james-project/blob/master/dockerfiles/run/guice/cassandra-ldap/destination/conf/webadmin.properties`